src/test/mir-opt/inline/inline-into-box-place.rs

```diff
@@ -1,6 +1,6 @@
 // ignore-wasm32-bare compiled with panic=abort by default
 // compile-flags: -Z mir-opt-level=3
-// only-64bit FIXME: the mir representation of RawVec depends on ptr size
+// EMIT_MIR_FOR_EACH_BIT_WIDTH
```

Comment:
Is this already implemented somewhere in a prior PR?

Can we also avoid shouting and use the same snake-case as is used for other compiletest options?

Reply:
The annotation is from #69916 which was copying the style of the old `mir-opt` tests.

I've only added it here to get rid of the stray `only-64bit` (all other `mir-opt` tests that have platform differences use `// EMIT_MIR_FOR_EACH_BIT_WIDTH` already)

Reply:
Ah, okay, then fine by me. I would like to see the shouting removed though, seems odd :)

Reply:
Yeah, I agree, I think the `compiletest` precedent should be stronger than the old `mir-opt` testing (which #70721 already removed). Maybe someone from @rust-lang/wg-mir-opt wants to  play around with this.

(The implementation is also ad-hoc, it doesn't use the standard `compiletest` directives, we'd want to fix that too).

Reply:
Also is there documentation somewhere on the mir-opt format? I checked rustc-dev-guide, did not find anything.